packages/api-server/README.md

```diff
@@ -0,0 +1,29 @@
+Redwood [Fastify](https://www.fastify.io) Server
+
+## package.json Server Binaries
+
+### `rw-serve`
+Not intended for production use.
```

Comment:
I think most people will find this scary. I think we need to qualify this statement a bit more - its totally fine to use in production, if you're using pm2/kube/etc. or if you don't have lots of concurrent requests.

Reply:
@dac09 Is your comment in the context of `rw-serve` or just `rw-serve api`?

Reply:
How about:
> This command runs both the API and Web server on the same port and is not performant at scale for production use. Instead, use the commands to run the API and (if needed) Web servers separately, along with tools like PM2, Nginx, or Kubernetes, which appropriately handle concurrent requests, errors, static asset, etc. for production contexts.

Reply:
Took a stab via this commit: https://github.com/redwoodjs/redwood/pull/4278/commits/0e785a6294226c72d0eca276f8c81ac3a62a5a96

Resolving conversation for now. Please re-open as needed.